### PR TITLE
Rework landing

### PR DESCRIPTION
**Pull request by timothymcmackin**
Rework the doc overview page to be a little more welcoming and start from the beginning instead of assuming that users know what to do with decentralized computing.

Preview: https://docs-staging-git-rework-landing-trili-tech.vercel.app/overview/

### CODE REVIEW

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:11]: I'd suggest either removing this header or putting it at the beginning of the page, since the previous paragraph also talk about what is Tezos.

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:22]: something wrong in "create and use digital currencies ... to use for digital currencies"

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:38]: "in development that will greatly enlarge": isn't this a bit outdated?
maybe replace by "that greatly enlarges" or so?

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:51]: "in addition to" seems incorrect, because minting and collecting are pointless without each other, so there are no two different things, but just two faces of the same thing - one for authors and one for users.
Replace by "and provide" or "and allow users"?

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:81]: forgot about voting others' proposals:
```suggestion
Anyone who holds XTZ — the chain's native token — can propose or vote about changes to how Tezos works, such as changes to gas fees and block times, new features such as Smart Rollups, or even major changes like how the consensus mechanism works.
```

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:89]: update for staking?
"can delegate tokens to a baker" -> "can delegate tokens to a baker or stake tokens on behalf of a baker"

Comment (NicNomadic) [reply docs/overview/index.md:89]: also "The bakers keep control of their tokens" -> "The bakers and stakers keep control of their tokens"

File: docs/overview/index.md
Comment (timothymcmackin) [reply docs/overview/index.md:11]: Removed.

File: docs/overview/index.md
Comment (timothymcmackin) [reply docs/overview/index.md:49]: Changed to "They allow creators to mint and sell their work in addition to providing a platform for users to collect such works."

Comment (timothymcmackin) [reply docs/overview/index.md:49]: Changed to "They provide a platform for creators to mint and sell their work and for users to collect such works."

File: docs/overview/index.md
Comment (timothymcmackin) [reply docs/overview/index.md:88]: Changed to "Users who want to participate without running a node themselves can delegate tokens to a baker or stake tokens on behalf of a baker for a share of the rewards. The bakers and stakers keep control of their tokens in that they can unstake them later; delegators keep complete control of their tokens."

File: docs/overview/index.md
Comment (timothymcmackin) [reply docs/overview/index.md]: Thanks, updated to "Users can create and use digital objects known as _tokens_ to use as digital currencies and to make it easier to accept online payments."

**Comment by timothymcmackin** [comment]
@NicNomadic Sorry, I forgot about this item. Would you please re-review? I'll fix conflicts in other PRs after this is merged.

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:20]:
```suggestion
- **Cryptocurrencies**: Users can create digital objects known as _tokens_ to be used as digital currencies and to make it easier to accept online payments.
```

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:86]: why short?
```suggestion
Users who want to participate without running a node themselves can delegate and stake tokens to a baker for a share of the baker's rewards. Users retain full control of their delegated and staked tokens: delegated tokens remain always liquid, and staked tokens are frozen but remain in the user's account and can be unstaked at any point to make them liquid again after a short delay.
```

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:89]:
```suggestion
The proof-of-stake model improves scalability and ensure cooperation via incentives. It also increases the cost of 51% attacks and avoids environmentally wasteful proof-of-work. Tezos launched in June 2018 as one of the first major proof-of-stake networks.
```

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:105]:
```suggestion
To start using and developing on Tezos, see [Tutorials](../tutorials).
```

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:107]:
```suggestion
For more details on installing, using, or contributing, to the platform, see the Octez documentation at https://tezos.gitlab.io/index.html.

For interactive learning material (including exercises) about Tezos concepts, the underlying technology, application development, and the ecosystem, see https://opentezos.com.
```

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:103]: I think this section should be moved near the top of the page, re-titled something like "Scope of this website", and start with the scope of the current documentation, not just sending to other websites, because as presented now, readers may ask themselves "what is the following of *this* for?".

Comment (timothymcmackin) [reply docs/overview/index.md]: Reworked  the section to be at the top and to say "Scope of this site and other Tezos documentation."

File: docs/overview/index.md
Comment (NicNomadic) [inline docs/overview/index.md:21]: Let's state the audience:
```suggestion
- A presentation of the main Tezos concepts and features for technical users, including developers and bakers
```

File: docs/overview/index.md
Comment (NicNomadic) [reply docs/overview/index.md]: Fine, thanks!

**Review by NicNomadic — DISMISSED** [review]
LGTM modulo one last thread

**Review by NicNomadic — APPROVED** [review]

**Pull request merged**
